wezterm-font/src/shaper/harfbuzz.rs

```diff
@@ -682,12 +682,14 @@ mod test {
 
     #[test]
     fn ligatures() {
+        config::use_test_configuration();
```

Comment:
with the suggested change to include the vendored fonts in test mode, the changes to this file can be reverted